src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/PublishContent.targets

```diff
@@ -25,7 +25,9 @@
     <CreateAzureContainer
       AccountKey="$(CloudDropAccessToken)"
       AccountName="$(CloudDropAccountName)"
-      ContainerName="$(ContainerName)" />
+      ContainerName="$(ContainerName)"
+      WriteOnlyTokenDaysValid="1"
+      ReadOnlyTokenDaysValid="1" />
```

Comment:
Sounds good.  It wasn't clear to me where the previous refactoring had occurred.  I'll remove this change from my PR since I don't want you to have to deal with coordinating our PR's or tracking down another place to make a change.